exercises/sub-list/SubListTest.swift

```diff
@@ -0,0 +1,95 @@
+import XCTest
+
+class SubListTest: XCTestCase {
+
+    func testEmptyLists() {
+
+        XCTAssertEqual(SublistKind.equal, classifier(listOne: [], listTwo: []) )
+    }
+
+    func testEmptyListWithinNonEmptyList() {
+
+        XCTAssertEqual(SublistKind.sublist, classifier(listOne: [], listTwo: [1, 2, 3]))
```

Comment:
You can also drop the first part of the enum `XCTAssertEqual(.sublist, ...`
This is up to you. 
I like leaving out the first part because it doesn't force the implementor to use the same name for their enum.